projects/ngx-formentry/src/components/ngx-remote-select/ngx-remote-select.component.ts

```diff
@@ -138,17 +138,22 @@ export class RemoteSelectComponent implements OnInit, ControlValueAccessor {
 
```

Comment:
It might be useful to add a cleanup function to prevent any potential memory leaks:

```ts
  ngOnDestroy() {
    this.remoteOptionInput$.complete();
  }
```